src/components/Tokens/TokenDetails/MobileBalanceSummaryFooter.tsx

```diff
@@ -86,6 +87,7 @@ export default function MobileBalanceSummaryFooter({ token }: { token: Currency
   const formattedBalance = formatCurrencyAmount(balance, NumberType.TokenNonTx)
   const formattedUsdValue = formatCurrencyAmount(useStablecoinValue(balance), NumberType.FiatTokenStats)
   const chain = CHAIN_ID_TO_BACKEND_NAME[token.chainId].toLowerCase()
+  const { value: enabled } = useGate('web_dummy_gate_amplitude_id')
```

Comment:
I would recommend adding a feature flag constants file where the gate names are added, and they can be referenced from. 

Would provide a good example for future feature flags + see them all in one place. 

something like `featureFlags.ts`:

```
export enum FEATURE_FLAGS {
  DummyFlag = 'web_dummy_gate_amplitude_id',
  DummyFlag2 = `web_dummy_gate_2_amplitude_id`
}
```

Reply:
suggestion taken